WebApplication/Startup.cs

```diff
@@ -61,15 +61,19 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<Arranger>();
             services.AddTransient<ProjectWork>();
             services.AddTransient<DtoGenerator>();
-            services.AddSingleton<DesignAutomationClient>(provider =>
+            services.AddSingleton<DesignAutomationClient>((provider) =>
                                     {
                                         var forge = provider.GetService<IForgeOSS>();
                                         var httpMessageHandler = new ForgeHandler(Options.Create(forge.Configuration))
                                         {
                                             InnerHandler = new HttpClientHandler()
                                         };
                                         var forgeService = new ForgeService(new HttpClient(httpMessageHandler));
-                                        return new DesignAutomationClient(forgeService);
+
+                                        var rsdkCfg = Configuration.GetSection("DesignAutomation").Get<Configuration>();
```

Comment:
IIRC - if `DesignAutomation` will be "level up" then it will be read automatically:
https://github.com/Autodesk-Forge/forge-configurator-inventor/blob/92474d3255a7f05a65cc64dd70bc84d6dbd18bdf/WebApplication/appsettings.Local.template.json#L6-L8

I mean - it should be on the same level as `Forge`.

Example:
```json
"Forge": {
    "ClientId": "xxxxx",
    "ClientSecret": "yyyyyy",
    "AuthenticationAddress": "https://developer-dev.api.autodesk.com/authentication/v1/authenticate"
},
"DesignAutomation": {
    "BaseAddress": "https://developer-dev.api.autodesk.com/da/us-west/"
}
```
@mivasi @hradilm

Reply:
@erymski this PR was just to talk about the problem with @vasicem, therefore it is closed no. Instead #401 was implemented

Reply:
And, in fact, in order to work `DesignAutomation` element had to be nested under `Forge`

Reply:
Yeah, seems you are right:
https://github.com/Autodesk-Forge/forge-api-dotnet-design.automation/blob/39861656ee879f1a53dce49832a9ae99fbd77372/src/Autodesk.Forge.DesignAutomation/ServiceCollectionExtensions.gen.cs#L39

Sorry for confusion, it was in my old notes, I guess I messed up something